vita3k/interface.cpp

```diff
@@ -115,12 +115,23 @@ bool install_vpk(HostState &host, GuiState &gui, const fs::path &path) {
     }
 
     SfoFile sfo_handle;
+    std::string content_id;
     sfo::load(sfo_handle, params);
     sfo::get_data_by_key(host.io.title_id, sfo_handle, "TITLE_ID");
     sfo::get_data_by_key(host.game_title, sfo_handle, "TITLE");
     sfo::get_data_by_key(host.game_version, sfo_handle, "APP_VER");
+    sfo::get_data_by_key(host.game_category, sfo_handle, "CATEGORY");
+    sfo::get_data_by_key(content_id, sfo_handle, "CONTENT_ID");
+    content_id = content_id.substr(20);
```

Comment:
What does `20` mean here? Try to give it a variable name or explain it in the comments.
`constexpr size_t startOfMessage = 20;` or something